ui/components/app/modals/modal.js

```diff
@@ -140,6 +141,33 @@ const MODALS = {
     },
   },
 
+  ADD_NEW_CONTACT: {
```

Comment:
We should not be using the Modal component or adding new modals to this older Modal system. Instead we have a <Popover> component that is used in a number of places. Modal is planned for deprecation.